python-libraries/nanover-ase/tests/test_imd_calculator.py

```diff
@@ -126,6 +133,8 @@ def test_one_interaction(
         with client_interaction(imd_client, interact_c):
             time.sleep(0.1)
             assert len(imd_calculator.interactions) == 1
+            # Update interactions in force manager
+            imd_calculator._imd_force_manager.update_interactions()
```

Comment:
This suggests that `update_interactions` should be available as a method on `imd_calculator` too -- without the caller having to be aware of the existence of the force manager.

Reply:
Perhaps also `add_to_frame_data` and then forget the `ImdForceManager` exists after construction and utilise it only via the `ImdCalculator` which it has been given to.